packages/composite-checkout-wpcom/src/components/wp-checkout-order-review.js

```diff
@@ -25,17 +25,13 @@ export default function WPCheckoutOrderReview( { className, removeItem } ) {
 	return (
 		<div className={ joinClasses( [ className, 'checkout-review-order' ] ) }>
 			<WPOrderReviewSection>
-				<WPOrderReviewLineItems
-					items={ items }
-					hasDeleteButtons={ true }
```

Comment:
There doesn't seem to be a case where `WPOrderReviewLineItems` is used without `hasDeleteButtons`, so I've just removed it.